typegate/src/runtimes/deno/deno_messenger.ts

```diff
@@ -28,10 +28,10 @@ export class DenoMessenger extends LazyAsyncMessenger<Worker, Task, unknown> {
               hrtime: false,
               net: true,
               // on request permissions
+              // read: ["tmp"], // default read permission
```

Comment:
let's put it `read: false` so it is disallowed if not overridden and we are safe if we forget it